<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="des_x" default="6" />
    <arg name="des_y" default="2" />
    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />

    <node name="follow_wall_server" pkg="crazyCmd" type="dd_follow_wall_server.py" output="screen"/>
    <node name="go_to_point_server" pkg="crazyCmd" type="dd_go_to_point_server.py" output="screen"/>
    <node name="bug0" pkg="crazyCmd" type="dd_bug0.py" output="screen"/>
</launch>